calcom/cal.com
# fix(a11y): added description for icon buttons

## What does this PR do?

the icons to switch in between monthly, weekly and column view aren't providing any textual description that is e.g. relevant for screenreaders. The included self developed tooltip doesn't work as a workaround for this.



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



You could for example check the accessibility sub-tab after selecting the `button`-element within Google Developer Tools to check for the included description, see the following screenshot with the current result and missing description:
<img width="1272" alt="image" src="https://github.com/user-attachments/assets/e3a7ba14-ad20-4b90-b65c-f2695774e29f">

## Checklist



- I haven't checked if my changes generate no new warnings
